cloud/cloud_env_impl.h

```diff
@@ -218,13 +218,22 @@ class CloudEnvImpl : public CloudEnv {
   // Files both in S3 and in the local directory have this [epoch] suffix.
   std::string RemapFilename(const std::string& logical_path) const override;
 
-  // This will delete all files in dest bucket and locally whose epochs are
-  // invalid. For example, if we find 00010.sst-[epochX], but the real mapping
-  // for 00010.sst is [epochY], in this function we will delete
-  // 00010.sst-[epochX]. Note that local files are deleted immediately, while
-  // cloud files are deleted with a delay of one hour (just to prevent issues
-  // from two RocksDB databases running on the same bucket for a short time).
-  Status DeleteInvisibleFiles(const std::string& dbname);
+  // This will delete all files in dest bucket and locally, which don't belong
+  // to CLOUDMANIFEST-cookie.
+  // - If cookie is empty, only MANIFEST file and sst
+  // files will be deleted.
+  // - If cookie is not empty, CLOUDMANIFEST files with
+  // different cookie will be deleted as well
+  //
+  // MANIFEST file and sst files will be deleted if their epoch is not current
+  // epoch of CLOUDMANIFEST-cookie. For example, if we find 00010.sst-[epochX],
+  // but the real mapping for 00010.sst is [epochY], in this function we will
+  // delete 00010.sst-[epochX].
+
+  // Note that local files are deleted immediately, while cloud files are
+  // deleted with a delay of one hour (just to prevent issues from two RocksDB
```

Comment:
Is the one hour delay of deleting cloud files something that's already implemented in RocksDB cloud? Just curious because I don't think I see that specific change here

Reply:
yes. It's already implemented: https://github.com/rockset/rocksdb-cloud/blob/master/cloud/cloud_env_impl.h#L422